plugins/obs-ffmpeg/obs-ffmpeg.c

```diff
@@ -165,6 +167,59 @@ static bool nvenc_device_available(void)
 extern bool load_nvenc_lib(void);
```

Comment:
nit: put the linux code above this forward declaration. The forward declaration is needed for `nvenc_supported` so lets keep it close to that if we are not going to merge the `#ifdef _WIN32` blocks.

Reply:
Done.